charts/shoot-core/components/charts/vpn-shoot/templates/deployment.yaml

```
@@ -5,7 +5,7 @@ metadata:
   name: vpn-shoot
   namespace: kube-system
   labels:
-    garden.sapcloud.io/role: system-component
+    gardener.cloud/role: system-component
```

The label selector for the healthchecks is still using the `garden.sapcloud.io/role`.
Ref https://github.com/gardener/gardener/blob/9d0fdfc183deb455b84af9cf94cc94471c964736/pkg/operation/botanist/health_check.go#L52-L64

```suggestion
    garden.sapcloud.io/role: system-component
```

---

Okay, added it in addition to the new label, when all shoots are on a supported version we can remove the old one.